src/app/static/src/app/components/downloadResults/Download.vue

```diff
@@ -44,10 +45,16 @@
                 type: Boolean
             }
         },
-        methods: {
-            download() {
-                this.$emit("trigger-download")
-            }
+        setup(props) {
+            const download = () => {
+                const fileUrl = `/download/result/${props.file.downloadId}`
+                const link = document.createElement("a");
+                link.href = fileUrl;
+                link.download = props.file.name || "downloaded_file";
```

Comment:
I don't think `props.file.name` is ever set here? So I think fine to hard code this